### PR TITLE
improve weather icons

### PR DESCRIPTION
**Pull request by JosephShepin**
- Added cleaner icons for weather
- Removed vue-ionicons 
Screenshot 2023-12-27 at 11 46 52 AM

### CODE REVIEW

File: src/components/cards/WeatherCard.vue [hunk 1]
```diff
@@ -164,36 +157,21 @@ export default {
...
         position: absolute
-        left: 78%
+        left: 85%
         transform: translate(-50%, 0)
         display: grid
         place-items: center
 
         .condition-icon
-          font-size: 2em
-          color: var(--color)
```
Comment (aw-0) [inline src/components/cards/WeatherCard.vue:174]: Any reason why you removed colors in the icons? Feel like it helped make the rest of the widget more coherent with the site.

**Review by aw-0 — COMMENTED** [review]
Good idea to remove the extra deps and move to our own svg icons.

Curious about the removal of colors but LGTM 👍

File: src/components/cards/WeatherCard.vue (hunk 1 above)
Comment (JosephShepin) [reply src/components/cards/WeatherCard.vue:174]: good catch, i'll add those back. SVG colors work weirdly so i'll have to do some research to fix this.

Comment (JosephShepin) [reply src/components/cards/WeatherCard.vue:174]: Screenshot 2023-12-28 at 5 41 52 PM

**Pull request merged**